docs/considerations.rst

```diff
@@ -0,0 +1,214 @@
+In this example, the extra 14ms of acquisition time per RF pulse is the cost of multi-echo fMRI.
+
+One way to think about this cost is in comparison to single-echo fMRI.
+If a multi-echo sequence has identical spatial resolution and acceleration as a single-echo sequence,
+then a rough rule of thumb is that the multi-echo sequence will have 10% fewer slices or 10% longer TR.
+Instead of compromising on slice coverage or TR, one can increase acceleration.
+If one increases acceleration, it is worth doing an empirical comparison to make sure there
+isn't a non-trivial loss in SNR or an increase of artifacts.
+
+Weighted Averaging may lead to an increase in SNR 
+-------------------------------------------------
+Multiple studies have shown that a
+weighted average of the echoes to optimize T2* weighting, sometimes called "optimally combined,"
+gives a reliable, modest boost in data quality. The optimal combination of echoes can currently be
+calculated in several software packages including AFNI, fMRIPrep, and tedana. In tedana, the weighted
+average can be calculated with `t2smap`_ If no other
+acquisition compromises are necessary to acquire multi-echo data, this boost is worthwhile. 
+
+Consider the life of the dataset
+--------------------------------
+If other
+compromises are necessary, consider the life of the data set. If data is being acquired for a discrete
+study that will be acquired, analyzed, and published in a year or two, it might not be worth making
+compromises to acquire multi-echo data. If a data set is expected to be used for future analyses in later
+years, it is likely that more powerful approaches to multi-echo denoising will sufficiently mature and add
+even more value to a data set.
+
+Other multi-echo denoising methods, such as MEICA, the predecessor to tedana, have shown the potential for
+much greater data quality improvements, as well as the ability to more accurately separate visually similar
+signal vs noise, such as scanner based drifts vs slow changes in BOLD signal. These more powerful methods are
+still being improved, and the algorithms are still changing. Users need to have the time and knowledge to look
+at the denoising output from every run to make sure denoising worked as intended. 
+
+You may recover signal in areas affected by dropout
+---------------------------------------------------
+Typical single echo fMRI uses an echo time that is appropriate for signal across most of the brain. While this is effective
+it also leads to drop out in regions with low :math:T_2^* values. This can lead to low or even no signal at all in some areas. 
+If your research question could benefit from having either
+improved signal characteristics in regions such as the orbitofrontal cortex, ventral temporal cortex or 
+the ventral striatum them multi-echo fMRI may be beneficial. 
+
+Consider the cost of added quality control
+------------------------------------------
+If someone wants a push-button
```

Comment:
I'd add a disclaimer here that _all data should be QC'ed_ !

Reply:
ALWAYS LOOK AT YOUR DATA!!!!

Reply:
to be clear, this is me agreeing.